mne/viz/misc.py

```diff
@@ -559,7 +546,7 @@ def plot_bem(subject=None, subjects_dir=None, orientation='coronal',
 
     # Plot the contours
     return _plot_mri_contours(mri_fname, surfaces, src, orientation, slices,
-                              show, show_indices, show_orientation)[0]
+                              show, show_indices, show_orientation)
```

Comment:
Did you `git grep _plot_mri_contours` to make sure no other returns need to be changed? IIRC this is shared across BEM plotting and dipole plotting

Reply:
Yeah I had to change the one in `report.py` but that was it